src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -1280,6 +1280,92 @@ void verifyDiagnosticsWithSource(string source, (Diagnostic, TextSpan)[] reportD
             }
         }
 
+        [Fact, WorkItem(66337, "https://github.com/dotnet/roslyn/issues/66337")]
+        public void Diagnostics_Respect_SuppressMessageAttribute()
```

Comment:
Can we get the same tests for VB?

Reply:
Didn't notice there are VB tests for this. Added.